ember-app/app/models/repo.js

```diff
@@ -110,6 +110,12 @@ var Repo = Ember.Object.extend(Serializable,{
     return Ember.$.post(api, {
       link: name,
     },'json');
+  },
+  fetchCommits: function(repo){
```

Comment:
@rauhryan I left this method in simply because it may prove useful in the future.